jest_setup.js

```diff
@@ -0,0 +1,17 @@
+// Jetpack mocks
+global.window.JP_CONNECTION_INITIAL_STATE = {};
+
+jest.mock( './jetpack/projects/js-packages/config/src', () => ( {
+	__esModule: true,
+} ) );
+
+jest.mock(
```

Comment:
After the Jest 29 update, there is an issue where it can't find the native export of this file so I added a mock.